packages/@aws-cdk/core/lib/stack-synthesizers/default-synthesizer.ts

```diff
@@ -202,6 +210,11 @@ export class DefaultStackSynthesizer extends StackSynthesizer {
    */
   public static readonly DEFAULT_FILE_ASSET_KEY_ARN_EXPORT_NAME = 'CdkBootstrap-${Qualifier}-FileAssetKeyArn';
 
+  /**
+   * Default file asset prefix
+   */
+  public static readonly DEFAULT_FILE_ASSET_PREFIX = '${AWS::AccountId}';
```

Comment:
We didn't really discuss the target customer base or rollout plan. For now, I would feel more comfortable if the default was `''`, or the empty string.

Since you are using a nonstandard bucket, you already need to tell your customers to specify the stack synthesizer in a specific way (or you have a custom synthesizer), so you might as well keep this to your specific target audience.

Also, I would expect the prefix to end in a `/`, to make it an actual prefix. Otherwise, call it something like `folder`.